src/components/ConnectedFilter/utils.js

```diff
@@ -28,8 +28,13 @@ const getSingleFilterOption = (histogramResult, initHistogramRes) => {
     });
     return rangeOptions;
   }
-
-  const textOptions = histogramResult.histogram.map((item) => ({
+  let rawtextOptions = histogramResult.histogram;
+  // hide guppyConfig.explorerHideDataFilter from filters
+  if (guppyConfig && guppyConfig.explorerHideDataFilter) {
+    rawtextOptions = histogramResult.histogram
+      .filter((item) => item.key !== guppyConfig.explorerHideDataFilter);
```

Comment:
what is the content of this `explorerHideDataFilter`? can you put some explainations in comments or docs?

Reply:
is there a place in guppy to better explain this? was going to put a better explication in data portal where the flag is set

Reply:
that is fine, just get a portal PR and explain in there, reference that PR so we can cross check

Reply:
Other PR https://github.com/uc-cdis/data-portal/pull/837